snow/engine/snowman/bootstrap/bootstrapper_test.go

```diff
@@ -1510,3 +1502,127 @@ func TestBootstrapContinueAfterHalt(t *testing.T) {
 		t.Fatal("Should have left blk1 as missing")
 	}
 }
+
+func TestBootstrapNoParseOnNew(t *testing.T) {
+	require := require.New(t)
+
+	ctx := snow.DefaultConsensusContextTest()
+
+	peers := validators.NewSet()
+
+	sender := &common.SenderTest{}
+	vm := &block.TestVM{}
+
+	sender.T = t
+	vm.T = t
+
+	sender.Default(true)
+	vm.Default(true)
+
+	isBootstrapped := false
+	bootstrapTracker := &common.BootstrapTrackerTest{
+		T: t,
+		IsBootstrappedF: func() bool {
+			return isBootstrapped
+		},
+		BootstrappedF: func(ids.ID) {
+			isBootstrapped = true
+		},
+	}
+
+	sender.CantSendGetAcceptedFrontier = false
+
+	peer := ids.GenerateTestNodeID()
+	if err := peers.Add(peer, nil, ids.Empty, 1); err != nil {
+		t.Fatal(err)
+	}
+
+	peerTracker := tracker.NewPeers()
+	startupTracker := tracker.NewStartup(peerTracker, peers.Weight()/2+1)
+	peers.RegisterCallbackListener(startupTracker)
+
+	if err := startupTracker.Connected(context.Background(), peer, version.CurrentApp); err != nil {
+		t.Fatal(err)
+	}
+
+	commonConfig := common.Config{
+		Ctx:                            ctx,
+		Beacons:                        peers,
+		SampleK:                        peers.Len(),
+		Alpha:                          peers.Weight()/2 + 1,
+		StartupTracker:                 startupTracker,
+		Sender:                         sender,
+		BootstrapTracker:               bootstrapTracker,
+		Timer:                          &common.TimerTest{},
+		AncestorsMaxContainersSent:     2000,
+		AncestorsMaxContainersReceived: 2000,
+		SharedCfg:                      &common.SharedConfig{},
+	}
+
+	snowGetHandler, err := getter.New(vm, commonConfig)
```

Comment:
```suggestion
	snowGetHandler, err := getter.New(vm, commonConfig)
	require.NoError(err)
```

Reply:
Ah you beated me to it

Reply:
Switched to use `require` everywhere in this test.